playbooks/bootstrap_cluster_teardown.yml

```diff
@@ -0,0 +1,8 @@
+- hosts: local
```

Comment:
Noticed some issues in another PR when running playbooks on tower using the `local` hostname. Might be better to rename this to `localhost` instead so that it fits into our existing inventory configurations